src/_sass/_variables.scss

```diff
@@ -1,42 +1,63 @@
-// Use of !important is to ensure overriding CA State Template values
-// some of which are more difficult than others.
+// Variables
 
-$btn-white-space: nowrap;
-
-$card-border-radius: 12px !important;
-
-$font-family: "Poppins", sans-serif !important;
-$font-family-sans-serif: $font-family;
 $headings-font-family: "Raleway", sans-serif !important;
```

Comment:
I wonder if we can get rid of these `!important;` declarations, as noted from the removed comment, they were mostly to ensure overrides of the CA State Template worked as expected.